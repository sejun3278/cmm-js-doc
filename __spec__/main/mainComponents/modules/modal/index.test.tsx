import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";

import MyModal from "src/main/mainComponents/modules/modal";

export default describe("Modal Module Page", () => {
  // 스냅샷
  test("Modal Module Page - Snapshot", () => {
    const { container } = render(
      <RecoilRoot>
        <MyModal />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
