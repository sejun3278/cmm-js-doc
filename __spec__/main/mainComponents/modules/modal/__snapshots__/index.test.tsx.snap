// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Module Page Modal Module Page - Snapshot 1`] = `
<div>
  <main
    class="_main_template_wrapper_ css-1p2vox9"
  >
    <section
      class="_main_title_wrapper_ css-j3xf1"
    >
      <div>
        <h1
          class="_title_ _main_title_"
        />
      </div>
      <p
        class="_p_ _main_title_remarks_ css-pzlb5d"
      />
    </section>
    <div
      class="css-1po5u6t"
    >
      <div
        class="_subTitleTemplate_ _howUse_Title_ css-k59gj9"
      >
        <div>
          <h2
            class="_title_ _subTitle_"
          />
        </div>
        <div
          class="_subTitleLine_ css-18klhw4"
        />
        <div
          class="css-1m3zms3"
          role="code-wrapper"
        >
          <p
            class="_p_ _howUse_example_notice_ css-pzlb5d"
          >
            기본값으로 사용할 수 있는 예시 코드입니다.
          </p>
          <div
            class="_copy_ css-que9y8"
          >
            <div
              class="css-eppsv9"
            >
              <pre
                class="css-1gdhu95"
              >
                <code>
                  <span
                    class="purple"
                  >
                    import
                  </span>
                  <span
                    class="yellow"
                  >
                     { 
                  </span>
                  <span
                    class="skyblue"
                  >
                    _
                  </span>
                  <span
                    class="yellow"
                  >
                     } 
                  </span>
                  <span
                    class="purple"
                  >
                    from
                  </span>
                  <span
                    class="lightOrange"
                  >
                     "mcm-js"
                  </span>
                  <span
                    class="sub"
                  >
                    ;
                  </span>
                </code>
              </pre>
            </div>
            <button
              class="css-12vdm2a"
            >
              <img
                class="_image_ _copyIcon_ css-msby2z"
                src="/images/commons/icons/copy.png"
              />
            </button>
          </div>
        </div>
      </div>
      <div
        class="_subTitleTemplate_ _example_template_title_ css-k59gj9"
      >
        <div>
          <h2
            class="_title_ _subTitle_"
          />
        </div>
        <div
          class="_subTitleLine_ css-18klhw4"
        />
        <div
          class="css-1sw7z4p"
        >
          <div
            class="css-bqybxg"
          >
            <div>
              <h3
                class="_title_"
              />
            </div>
            <div
              class="css-1bzs2hq"
            />
          </div>
          <div
            class="css-bqybxg"
          >
            <div>
              <h3
                class="_title_"
              />
            </div>
            <div
              class="css-1bzs2hq"
            >
              0
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
